config/hydroshare-config.yaml

```diff
@@ -33,16 +33,16 @@ IRODS_LOCAL_MOUNT: /irods
 ### These give Nginx direct access to iRODS vaults
 ### All ROOTs must be children of DOCKER_MAP ###
 IRODS_DOCKER_MAP: /irods
-IRODS_DATA_ROOT: /irods/data
-IRODS_USER_ROOT: /irods/user
+IRODS_DATA_ROOT: /irods/data/iRODS/Vault/home/wwwHydroProxy
+IRODS_USER_ROOT: /irods/user/iRODS/Vault/home/localHydroProxy
 IRODS_CACHE_ROOT: /irods/cache
```

Comment:
do these overlap with settings.py and local_settings.py?

Reply:
yes, that's settings on data.local.org and users.local.org. If dev knows how to config/modify that means he/she no needs this PR